src/util/factory.js

```diff
@@ -228,10 +228,10 @@ const GoogleSheetInput = function () {
     var queryString = window.location.href.match(/sheetId(.*)/)
     var queryParams = queryString ? QueryParams(queryString[0]) : {}
 
-    if (domainName && queryParams.sheetId.endsWith('.csv')) {
```

Comment:
This will create error with our home page. Instead `queryParams.sheetId && queryParams.sheetId.endsWith('.csv')` condition can help